front-end/package.json

```diff
@@ -29,38 +31,22 @@
     "@angular/platform-browser": "~13.2.0",
     "@angular/platform-browser-dynamic": "~13.2.0",
     "@angular/router": "~13.2.0",
-    "@fortawesome/fontawesome-free": "~5.15.4",
-    "@kolkov/angular-editor": "~2.0.0",
-    "@ng-bootstrap/ng-bootstrap": "~11.0.0",
-    "@ng-select/ng-select": "~8.1.1",
+    "@ngrx/effects": "^13.0.2",
+    "@ngrx/store": "^13.0.2",
+    "@ngrx/store-devtools": "^13.0.2",
```

Comment:
should this be a dev dependancy?

Reply:
I thought so too but the install instructions had a reason and also a setting so that it wasn't active in production. However, it doesn't seem to be working that well. I noticed that it isn't keeping up in the chome UI plugin what is actually in the store. I installed the "hydration" package that syncs the store with the browser localStorage and that is working great and allows better tracking. I am going to remove the store-devtools and push again